packages/hash/frontend/src/components/RemoteBlock/RemoteBlock.tsx

```diff
@@ -83,6 +84,14 @@ export const RemoteBlock: FunctionComponent<RemoteBlockProps> = ({
     }
   }, [graphProperties.linkedAggregations, graphService]);
 
+  useEffect(() => {
+    if (graphService) {
+      graphService.readonly({
+        data: graphProperties.readonly,
+      });
+    }
+  }, [graphProperties.readonly, graphService]);
```

Comment:
@nathggns do you see any better ways of syncing context with graph service (and ProseMirror too)? If the is one, we can implement it in a follow-up.

Reply:
What problems do you see with this approach @kachkaev ?

Reply:
`useEffect` creates a small time gap between a change in `graphProperties.readonly` and `graphService` state. This _might_ cause an ‘impossible’ app state and cause a crash. I don’t know enough to be sure one way or the other, so curious what others think. We can leave this as is for now I guess.